provisioning/ansible/roles/grc/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Install grc
+  apt:
+    name: grc
+    state: present
+    update_cache: yes
```

Comment:
Lastly, please pin a version of grc (just take the latest one). You can do so by writing
```yml
name: grc={{ grc_version }}
```
and then declaring the latter as a variable in `grc/defaults/main.yml`:
```yml
grc_version: 1.2.3.4
```

Reply:
Thanks for the guidance. I now pinned the version to the latest one.